datadog_checks_base/tests/test_openmetrics.py

```diff
@@ -2095,3 +2095,20 @@ def test_metadata_transformer(mocked_openmetrics_check_factory, text_data):
         m.assert_any_call('test:123', 'version.raw', 'v1.6.0-alpha.0.680+3872cb93abf948-dirty')
         m.assert_any_call('test:123', 'version.scheme', 'semver')
         assert m.call_count == 7
+
+
+def test_ssl_verify_not_raise_warning(mocked_openmetrics_check_factory, text_data):
+    instance = dict(
+        {
+            'prometheus_url': 'https://www.example.com',
+            'metrics': [{'foo': 'bar'}],
+            'namespace': 'openmetrics',
+            'ssl_verify': False,
+        }
+    )
+    check = mocked_openmetrics_check_factory(instance)
+    scraper_config = check.get_scraper_config(instance)
+
+    with pytest.warns(None):
```

Comment:
That's not enough to assert that the code doesn't display warnings. See https://doc.pytest.org/en/3.0.5/recwarn.html#recwarn.

Reply:
Good catch. Fixed with the latest commit.